src/components/Homepage/LendByCategory/LoanCategoriesSection.vue

```diff
@@ -1,8 +1,8 @@
 <template>
 	<div class="loan-category-section-wrapper">
 		<div class="category-options"
-			v-touch:swipe.left="scrollRowRight"
-			v-touch:swipe.right="scrollRowLeft"
+			v-touch:swipe.left="scrollCategoryNamesRight"
+			v-touch:swipe.right="scrollCategoryNamesLeft"
 			ref="categoryOptions"
 			:style="{ left: scrollPos + 'px' }"
 		>
```

Comment:
Github won't let me comment on the line itself, but regarding the `<a href="#" role="button">`s below - Since clicking the links doesn't take you to a new url, they should be button tags. We have a new class on `<kv-button class="text-link">` that can act as a starting point for styling.

Reply:
I think it'll pretty much be a dropin of `<kv-button class="text-link category-options__link" ...>`

Reply:
@ryan-ludwig thanks, I had that pending, I wasn't sure if the text-link class was added for kvButton yet